packages/components/src/Components/TreeTable/decorator.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Button } from '@patternfly/react-core';
```

Comment:
I think we could start to use full paths form PF-react and PF-icons.. however, not sure if it would be compatible with @Hyperkid123 's https://github.com/RedHatInsights/frontend-components/pull/340

Reply:
First, merge that PR. If you would start using absolute paths now, you would have more duplications.